components/kyma-operator/README.md

```diff
@@ -2,60 +2,65 @@
-  ```
-  sh ./hack/update-codegen.sh
-  ```
+### Use command-line flags
 
-## Build a Docker image
+| Name | Required | Description | Default value |
+|------|----------|-------------|-----------------|
+| **kubeconfig** | no | Path to a kubeconfig file | Taken from the `KUBECONFIG` env parameter|
+| **kymaDir** | no | Directory holding helm releases, within the image/pod | `/kyma` |
+| **backoffIntervals** | no | Number of seconds to wait before subsequent retries | `10,20,40,60,80` |
+| **overrideLogFile** | no | Log File to Print Installation overrides to. | `/dev/stdout` |
+| **overrideLogFormat** | no | Installation Override Log format (Accepted values: text or json) | `text` |
+| **helmMaxHistory**  | no | Max number of releases returned by Helm release history query | `10` |
+| **helmDriver** | no | Helm backed storage drivers. [Options and details](https://helm.sh/docs/helm/helm/#helm) | `secrets` |
+| **helmDebugMode** | no | Enable/Disable Helm client output in the Kyma Operator logs | `false` |
 
-Use the [`Makefile`](./Makefile) script to build a Docker image of the Kyma Operator:
-  ```
-  export DOCKER_PUSH_REPOSITORY={DOCKER_REPOSITORY}
-  make build-image
-  ```
+### Supported Environmental Variables
 
-## Details
+| Name | Default | Description |
+| ---- | ------- | ----------- |
+| **INST_NAMESPACE** | `default` | Namespace in which the Installation CR is located |
+| **INST_RESOURCE** | `kyma-installation` | Name of the Installation custom resource |
+| **INST_FINALIZER** | `finalizer.installer.kyma-project.io` | Name of the finalizer |
+| **OVERRIDES_NAMESPACE** | `kyma-installer` | Namespace in which the Installer overrides are located |
 
-### Upgrade Kyma
+## Custom Resource
 
-The upgrade procedure relies heavily on Helm. When you trigger the upgrade, Helm performs `helm upgrade` on Helm releases that exist in the cluster and are defined in the Kyma version you're upgrading to. If a Helm release is defined in the Kyma version you're upgrading to but is not present in the cluster when you trigger the upgrade, Helm performs `helm install` and installs such a release.
+The `installation.installer.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format the Kyma Operator Controller listens for. To get the up-to-date CRD and show
+the output in the `yaml` format, run this command:
 
-When you trigger Kyma upgrade, the Kyma Operator lists the Helm releases installed in your current Kyma version. This list is compared against the list of Helm releases of the Kyma version you're upgrading to. The releases are matched by their names. The releases that match between versions are upgraded through `helm upgrade`. Releases that don't match are treated as new and installed through `helm install`.
+```bash
+kubectl get crd installation.installer.kyma-project.io -o yaml
```

Comment:
Again the question: `installation.installer.kyma-project.io` or `installations.installer.kyma-project.io`?